plasmic.json

```diff
@@ -0,0 +1,621 @@
+{
+  "platform": "react",
+  "code": {
+    "lang": "ts",
+    "scheme": "blackbox"
+  },
+  "style": {
+    "scheme": "css",
+    "defaultStyleCssFilePath": "plasmic/plasmic__default_style.css"
+  },
+  "images": {
+    "scheme": "inlined"
+  },
+  "tokens": {
+    "scheme": "theo",
+    "tokensFilePath": "plasmic-tokens.theo.json"
+  },
+  "srcDir": "src/components",
+  "defaultPlasmicDir": "plasmic",
+  "projects": [
+    {
+      "projectId": "mSMs7FqGjFMPgGjdzmbmrV",
+      "projectName": "Venturemark",
+      "version": "latest",
+      "cssFilePath": "plasmic/venturemark/plasmic_venturemark.css",
+      "components": [
+        {
+          "id": "b40174b2-70e5-4c47-bb75-b9b1868ba709",
+          "name": "Landing",
+          "type": "managed",
+          "projectId": "mSMs7FqGjFMPgGjdzmbmrV",
+          "renderModuleFilePath": "plasmic/venturemark/PlasmicLanding.tsx",
+          "importSpec": {
+            "modulePath": "Landing.tsx"
+          },
+          "cssFilePath": "plasmic/venturemark/PlasmicLanding.css",
+          "scheme": "blackbox"
+        },
+        {
+          "id": "nqKXpmm91O",
+          "name": "LandingInput",
+          "type": "managed",
+          "projectId": "mSMs7FqGjFMPgGjdzmbmrV",
+          "renderModuleFilePath": "plasmic/venturemark/PlasmicLandingInput.tsx",
+          "importSpec": {
+            "modulePath": "LandingInput.tsx"
+          },
+          "cssFilePath": "plasmic/venturemark/PlasmicLandingInput.css",
+          "scheme": "blackbox"
+        },
+        {
+          "id": "xr8buS_I4PE",
+          "name": "AcceptInvite",
```

Comment:
Can we strip this down when we start out with it? I would like to drop everything we do not need so that we can add what we really need deliberately. That should result in us fully understanding how anything works. When we now add a hot mess of something we do not need and not fully comprehend we just go downhill again. I would like to just add what we need as discussed. We want to have some fake update and maybe flip it. We do not need invites or logins and all the other crap right now. Further, hundreds of files and tens of thousands of changes is nothing any human being can or want to review. ❤️